boefjes/boefjes/katalogus/dependencies/plugins.py

```diff
@@ -211,7 +215,8 @@ def get_pagination_parameters(offset: int = 0, limit: int | None = None) -> Pagi
 
 def get_plugins_filter_parameters(
     q: str | None = None,
+    ids: list[str] | None = Query(None),
```

Comment:
Is `Query` necessary here? Or will FastAPI assume it's a body field otherwise?

Reply:
Yeah its necessary...